imqsauth/http.go

```diff
@@ -721,7 +721,10 @@ func httpHandlerCreateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	}
 
 	// Get the userId of the logged-in user
-	createdby := r.token.UserId
+	var createdby authaus.UserId
+	if !strings.HasPrefix(r.http.Header.Get("Authorization"), "IMQS-Trusted ") {
```

Comment:
I think this would be better:
```go
if r.token != nil {
    createdby = r.token.UserId
}
```

Reply:
OK, agree. The called would have had to be authenticated with admin privileges (handlerFlagNeedAdminRights) higher up in the code, so updated.